packages/core/src/components/icon/icon.md

```diff
@@ -4,15 +4,62 @@
     See [**Icons**](#icons) for a searchable list of all available UI icons.
 </div>
 
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+    <h5>SVG icons in 2.0</h5>
+    Blueprint 2.0 introduced SVG icon support throughout the ecosystem, and moved icon resources to a separate
```

Comment:
nit: no need for the comma here (I noticed your prose style does this a lot and I generally don't mind for code comments, but I'm more picky about it in actual documentation).

Reply:
also, "throughout the ecosystem" is a bit of a stretch. I would remove that phrase entirely.

Reply:
sometimes, i use commas to indicate where to pause, for maximal dramatic effect. 🎭